nexus/types/src/deployment/execution/utils.rs

```diff
@@ -0,0 +1,228 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Should we call this file "dns"?  I think it's all DNS-specific?  (I'm always skeptical of things labeled "utils".)

Reply:
`Sled` and `blueprint_nexus_external_ips` are not DNS-specific. I'll move the DNS-specific ones into a `dns.rs`. The remaining don't have a good place to fit in, and I don't like putting code in `mod.rs` (because submodules get access to private fields in types) -- so would prefer `utils.rs` or something similar overall.

Reply:
Done.